docs/src/main/sphinx/admin/properties-node-scheduler.rst

```diff
@@ -30,17 +30,31 @@ due to splits not being balanced across workers. Ideally, it should be set
 such that there is always at least one split waiting to be processed, but
 not higher.
 
-``node-scheduler.max-pending-splits-per-task``
+``node-scheduler.min-pending-splits-per-task``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
 * **Type:** :ref:`prop-type-integer`
 * **Default value:** ``10``
 
-The number of outstanding splits with the standard split weight that can be
+The minimum number of outstanding splits with the standard split weight that can be
 queued for each worker node for a single stage of a query, even when the
 node is already at the limit for total number of splits. Allowing a minimum
 number of splits per stage is required to prevent starvation and deadlocks.
 
+This value must be smaller than ``node-scheduler.max-splits-per-node`` and
```

Comment:
this is currently not verified right?

Reply:
It is verified in factory (not in selector itself).